### PR TITLE
Use release for sdk harness tests.

### PR DESCRIPTION
**Pull request by adlerjohn**
Building Forc projects is very slow with debug, so turn on release.

### CODE REVIEW

**Review by nfurfaro — APPROVED** [review]
LGTM.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -315,12 +315,12 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: install
-          args: --debug --path ./forc
+          args: --release --path ./forc
```
Comment (adlerjohn) [inline .github/workflows/ci.yml:318]:
```suggestion
          args: --path ./forc
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -315,12 +315,12 @@ jobs:
...
       - name: Install Forc fmt
         uses: actions-rs/cargo@v1
         with:
           command: install
-          args: --debug --path ./forc-plugins/forc-fmt
+          args: --release --path ./forc-plugins/forc-fmt
```
Comment (adlerjohn) [inline .github/workflows/ci.yml:323]:
```suggestion
          args: --path ./forc-plugins/forc-fmt
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -332,7 +332,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --manifest-path ./test/src/sdk-harness/Cargo.toml -- --test-threads=1 --nocapture
+          args: --release --manifest-path ./test/src/sdk-harness/Cargo.toml -- --test-threads=1 --nocapture
```
Comment (adlerjohn) [inline .github/workflows/ci.yml:335]:
```suggestion
          args: --manifest-path ./test/src/sdk-harness/Cargo.toml -- --test-threads=1 --nocapture
```

**Review by nfurfaro — APPROVED** [review]

**Comment by adlerjohn** [comment]
This doesn't actually speed up building much

**Pull request closed without merging**